ember-bootstrap/addon/components/bs-form.js

```diff
@@ -97,7 +97,7 @@ import { ref } from 'ember-ref-bucket';
   It could be used for visual feedback about last submission:
 
   ```hbs
-  <BsForm @onSubmit={{action 'save}} as |form|>
+  <BsForm @onSubmit={{action "save"}} as |form|>
```

Comment:
Actually it should not use `{{action}}` helper anymore. But that could be done in a later refactoring. Especially as it affects more places.